app/service/data_svc.py

```diff
@@ -91,9 +91,9 @@ def __init__(self, dao, utility_svc):
             await self.dao.create('core_source_map', dict(op_id=op_id, source_id=s_id))
         return op_id
 
-    async def create_fact(self, property, value, source_id, score, blacklist, link_id=None):
+    async def create_fact(self, property, value, source_id, score, blacklist, group_id=None, link_id=None):
         await self.dao.create('core_fact', dict(property=property, value=value, source_id=source_id,
-                                                score=score, blacklist=blacklist, link_id=link_id))
```

Comment:
choose a different variable than group_id-- as it too closely resembles the "group" table. maybe use "set_id" instead?

Reply:
my mistake. Just changed it to set_id